web/frontend/.eslintrc.cjs

```diff
@@ -1,204 +1,47 @@
-// eslint-disable-next-line unicorn/prefer-module
+'use strict';
+
+const path = require('node:path');
+
 module.exports = {
   root: true,
-  env: {
-    browser: true,
-    es2021: true,
-  },
-  parser: "@typescript-eslint/parser",
+  extends: ['@viamrobotics/eslint-config/svelte'],
   parserOptions: {
-    ecmaVersion: 'latest',
-    sourceType: 'module',
-    project: './tsconfig.json',
-    extraFileExtensions: ['.svelte'],
     tsconfigRootDir: __dirname,
+    project: './tsconfig.json',
   },
-  plugins: [
-    '@typescript-eslint',
-    'unicorn',
-    'tailwindcss',
-    'promise',
-    'svelte',
-  ],
-  extends: [
-    'plugin:@typescript-eslint/recommended',
-    'eslint:all',
-    'plugin:unicorn/recommended',
-    'plugin:tailwindcss/recommended',
-    'plugin:svelte/recommended',
-    'plugin:svelte/prettier',
-    'plugin:promise/recommended',
-  ],
-  ignorePatterns: ['**/node_modules/**', '*.json', '**/protos/**', 'converter.js'],
-  overrides: [
-    {
-      files: ['*.ts'],
-      parser: '@typescript-eslint/parser',
-    }, {
-      files: ['*.svelte'],
-      parser: 'svelte-eslint-parser',
-      parserOptions: {
-        parser: '@typescript-eslint/parser',
-      },
-    }
-  ],
   settings: {
-    'import/extensions': ['.svelte'],
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts', '.js'],
-    },
-    'import/resolver': {
-      typescript: {
-        alwaysTryTypes: true,
-        project: './tsconfig.json',
-      },
-      'eslint-import-resolver-custom-alias': {
-        alias: {
-          '@': './frontend/src',
-        },
-        extensions: ['.ts', '.js', '.svelte'],
-      },
+    tailwindcss: {
+      config: path.join(__dirname, 'tailwind.config.ts'),
     },
   },
+  env: {
+    browser: true,
+    node: true,
+  },
   rules: {
-    // TODO(APP-2035): Promote back to error.
+    'no-void': ['error', { allowAsStatement: true }],
+
+    // @TODO(mp) These were disabled when moving over to js-config. They should be gradually re-enabled.
+    '@typescript-eslint/no-floating-promises': 'off',
+    '@typescript-eslint/no-misused-promises': 'off',
+    '@typescript-eslint/no-unsafe-enum-comparison': 'off',
     'svelte/valid-compile': 'off',
```

Comment:
Should they be `warn` rather than `off` if we intend to put them back to errors?

Reply:
Great point, updated